### PR TITLE
Check whether it is an expression before accessing the expression properties

### PR DESCRIPTION
**Pull request by SamvelRaja**
Need to check whether the node is a `expression` before getting the `expression properties`

Found while testing the issue #79 

Feel free to reject the PR if not needed.

### CODE REVIEW

File: lib/formulas/resource-router-mapping.js [hunk 1]
```diff
@@ -30,9 +30,11 @@ function traverseExpressions(routes) {
     if (isResourceRoute(route)) {
       transformResourceToRoute(route);
     }
-    var lastArgument = route.expression.arguments[route.expression.arguments.length-1];
-    if (types.FunctionExpression.check(lastArgument)) {
-      traverseExpressions(lastArgument.body.body);
+    if (route.expression) {
```
Comment (marcioj) [inline lib/formulas/resource-router-mapping.js]: Can you also check if the AST type is a `CallExpression` with `if (types.CallExpression.check(route.expression)) {` ?

**Comment by marcioj** [comment]
Looks good, can you add a test covering this issue?

File: lib/formulas/resource-router-mapping.js (hunk 1 above)
Comment (SamvelRaja) [inline lib/formulas/resource-router-mapping.js]: :+1:

**Comment by SamvelRaja** [comment]
Shall i include the test for normal route structure also? or this is enough? @marcioj

**Comment by marcioj** [comment]
> Shall i include the test for normal route structure also?

This is good to go, we have other tests with more common scenarios.

Thank you for your contribution!

**Comment by SamvelRaja** [comment]
:+1:

**Pull request merged**